notebook/notebookapp.py

```diff
@@ -645,6 +646,9 @@ def nbextensions_path(self):
             path.append(os.path.join(get_ipython_dir(), 'nbextensions'))
         return path
 
+    nbextensions = List(Unicode(), config=True,
+        help="A list of nbextensions to enable.")
+
```

Comment:
I don't think so. I want to keep the changes as minimal as possible. Also, this is likely the approach we will take in 5.0 when we move to npm packages.